packages/contractkit/test/driver.test.ts

```diff
@@ -0,0 +1,59 @@
+import { Address } from '../src/base'
+import { newKit } from '../src/kit'
+
+test('Test driver', async () => {
+  // Integration
+  // const kit = newKit('http://35.247.98.50:8545')
+  const kit = newKit('http://35.247.74.27:8545')
+
+  // kit.web3
+  const account: Address = '0xCEa3eF8e187490A9d85A1849D98412E5D27D1Bb3'
+  // const account2: Address = '0xCEa3eF5e187490A9d85A1849D98412E5D27D1Bb3'
+
+  // To interact with contract we obtain them from kit
+  const goldToken = await kit.contracts.getGoldToken()
+
+  const balance = await goldToken.balanceOf(account)
+  console.log(`balance:`, balance)
+
+  // // internally kit does:
```

Comment:
Why is this commented out?

Reply:
ignore this file... it's just an example and it would be removed later